src/components/trends/common/NoData.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import _ from 'lodash';
+import React, { PropTypes } from 'react';
+
+import styles from './NoData.css';
+
+const NoData = (props) => {
+  const noDataMessage = _.template('There is no <%= type %> data for this time period :(');
+  const { position, dataType } = props;
+
+  if (!position) {
+    return null;
+  }
+
+  return (
+    <text className={styles.noDataMsg} id="noDataMsg" x={position.x} y={position.y}>
+      {noDataMessage({ type: dataType })}
+    </text>
+  );
+};
+
+NoData.defaultProps = {
+  dataType: 'CBG',
```

Comment:
I think you may have misunderstood one of my previous comments. I don't think the *datatype* should be a default prop, but rather the message template itself as `messageTemplate` or similar (so we could override it as/if needed). Perhaps another default prop `displayTypes` can contain the user-facing datatype for each of the `cbg` and `smbg` types (maybe `{cbg: 'CGM', smbg: 'BGM'}`), but let's just pass in `cbg` or `smbg` as a required prop `type` (or `datatype` if React throws an error when you tried to use `type` - I can't remember if that might be a problem).

Also when I run locally I'm getting the default "CBG" even for BGM, so the active type is not being passed down.